tests/data/dataconverter/readers/mpes/config_file.json

```diff
@@ -14,328 +14,399 @@
+      "AXISNAME[trans_x]/@vector": [
+        1,
+        0,
+        0
+      ]
+    }
+  },
+  "/ENTRY[entry]/PROCESS[process]/DISTORTION[distortion]": {
+    "applied": true,
+    "symmetry": "@attrs:metadata/momentum_correction/rotsym",
+    "original_centre": "@attrs:metadata/momentum_correction/pcent",
+    "original_points": "@attrs:metadata/momentum_correction/pouter",
+    "cdeform_field": "@attrs:metadata/momentum_correction/cdeform_field",
+    "rdeform_field": "@attrs:metadata/momentum_correction/rdeform_field"
+  },
+  "/ENTRY[entry]/PROCESS[process]/REGISTRATION[registration]": {
+    "applied": true,
+    "depends_on": "/entry/process/registration/tranformations/rot_z",
+    "TRANSFORMATIONS[tranformations]": {
+      "AXISNAME[trans_x]": "@attrs:metadata/momentum_correction/adjust_params/xtrans",
+      "AXISNAME[trans_x]/@transformation_type": "translation",
+      "AXISNAME[trans_x]/@units": "pixels",
+      "AXISNAME[trans_x]/@vector": "@attrs:metadata/momentum_correction/adjust_params/x_vector",
+      "AXISNAME[trans_x]/@depends_on": ".",
+      "AXISNAME[trans_y]": "@attrs:metadata/momentum_correction/adjust_params/ytrans",
+      "AXISNAME[trans_y]/@units": "pixels",
+      "AXISNAME[trans_y]/@transformation_type": "translation",
+      "AXISNAME[trans_y]/@vector": "@attrs:metadata/momentum_correction/adjust_params/y_vector",
+      "AXISNAME[trans_y]/@depends_on": "trans_x",
+      "AXISNAME[rot_z]": "@attrs:metadata/momentum_correction/adjust_params/angle",
+      "AXISNAME[rot_z]/@units": "degrees",
+      "AXISNAME[rot_z]/@transformation_type": "rotation",
+      "AXISNAME[rot_z]/@offset": "@attrs:metadata/momentum_correction/adjust_params/offset",
+      "AXISNAME[rot_z]/@vector": "@attrs:metadata/momentum_correction/adjust_params/rotation_vector",
+      "AXISNAME[rot_z]/@depends_on": "trans_y"
+    }
+  },
+  "/ENTRY[entry]/PROCESS[process]/energy_calibration":{
+    "applied": true,
+    "coefficients": "@attrs:metadata/energy_correction/calibration/coeffs",
+    "fit_function": "@attrs:metadata/energy_correction/calibration/fit_function",
+    "original_axis": "@attrs:metadata/energy_correction/tof",
+    "calibrated_axis": "@attrs:metadata/energy_correction/calibration/axis"
+  },
+  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[kx_calibration]": {
+    "applied": true,
+    "scaling": "@attrs:metadata/momentum_correction/calibration/scale_kx",
+    "offset": "@attrs:metadata/momentum_correction/offset_kx",
+    "calibrated_axis": "@attrs:metadata/momentum_correction/calibration/axis_kx"
+  },
+  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[ky_calibration]": {
+    "applied": true,
+    "scaling": "@attrs:metadata/momentum_correction/calibration/scale_ky",
+    "offset": "@attrs:metadata/momentum_correction/offset_ky",
+    "calibrated_axis": "@attrs:metadata/momentum_correction/calibration/axis_ky"
+  },
+  "/ENTRY[entry]/data": {
+    "@axes": "@data:dims",
+    "AXISNAME_indices[@*_indices]": "@data:*.index",

```

Comment:
This seems to not be recognized by the parser. Does it need an additional @ in the beginning?

Reply:
For me it is properly created in the file:

<img width="480" alt="Bildschirmfoto 2024-01-19 um 10 13 11" src="https://github.com/FAIRmat-NFDI/pynxtools/assets/1014422/e4cd2f21-da78-4a62-8a8b-dca18a529b73">

Do you have a different output?

Reply:
No, it is working fine, I'm just wondering if this is related to the problems with recognizing the concept in the parser

Reply:
No, because the log file is only generated while reading the created nexus file. As long as it ends up correctly in the nexus file the `config_file.json` doesn't do anything in this debug log.